lib/modules/showImages.js

```diff
@@ -298,8 +307,21 @@ addModule('showImages', (module, moduleID) => {
 		const maximumTop = viewportHeight * (bufferScreens + 1);
 		const minimumBottom = viewportHeight * bufferScreens * -1;
 
-		imageList.forEach(image => {
-			if (image.imageLink && image.imageLink.media) {
+		activeImageList.forEach(image => {
```

Comment:
Nit: use `for (const image of activeImageList) {` when it's not chained with anything else. Also works with sets.